src/routes/configure.js

```diff
@@ -137,6 +138,9 @@ export default function Configure({ printJob, changePrintJob }) {
 
   return (
     <div>
+      <div>
+        <JobCard job={printJob} onSelectJob={()=>{}} img={printJob.thumbnail}/>
+      </div>
```

Comment:
I'm wondering, now that we have the job card showing on the configuration page, would it be possible for the thumbnail to change colors when the user selects a different color of material?